frontend/talentsearch/src/js/components/employmentEquityForm/dialogs/DisabilityDialog.tsx

```diff
@@ -68,25 +68,27 @@ const DisabilityDialog: React.FC<EquityDialogProps> = ({
     <Dialog
       isOpen={isOpen}
       onDismiss={onDismiss}
-      color="ts-primary"
+      color="ts-secondary"
       title={intl.formatMessage(getEmploymentEquityGroup("disability"))}
     >
       <UnderReview />
-      <p data-h2-margin="base(0, 0, x1, 0)">
+      <div data-h2-margin="base(x1, 0)">
```

Comment:
We could just add `data-h2-margin="base(x1, 0)"` to the `<p>` in `Definition.tsx` so we no longer need the wrapping `<div>` in each `Dialog`

Reply:
Oh, that's much cleaner! I'll rework it and resubmit the review.

Reply:
Sorry, I forgot you guys were trying to deploy this to Fable today!